### PR TITLE
Vs-eng-update-vine-attack-lv-383

### PR DESCRIPTION
**Pull request by RyanASwanson**
https://bradleycapstone.atlassian.net/jira/software/projects/LV/boards/32/backlog?assignee=6318bb796856bdd60aa045de&selectedIssue=LV-383
This task took a bit longer than intended (not as long as the github history may suggest, but still a bit) luckily the other attack update was very short so it balances out.

### CODE REVIEW

File: Assets/GameAssets/Boss/BossScripts/Attacks/Vine/VineAttackController.cs [hunk 1]
```diff
@@ -1,29 +1,20 @@
...
-// File Name :         FullRoomAttack.cs
+// File Name :         VineAttackController.cs
 // Author :            Tommy Roberts
 // Contributor :       Andrew Stapay
+//                     Ryan Swanson
 // Creation Date :     10/9/2024
 //
-// Brief Description : controls the full room attack for the boss
+// Brief Description : Controls the functionality for the bosses vine attack
 *****************************************************************************/
```
Comment (AlecatEnjoyer) [inline Assets/GameAssets/Boss/BossScripts/Attacks/Vine/VineAttackController.cs:9]: Couldn't this and VineAttack.cs be combined

**Review by AlecatEnjoyer — APPROVED** [review]
approved left suggestion

**Review by astapay — APPROVED** [review]
wahoo!

File: Assets/GameAssets/Boss/BossScripts/Attacks/AttackWarningZone.cs
Comment (ASLH68) [inline Assets/GameAssets/Boss/BossScripts/Attacks/AttackWarningZone.cs:1]: Add self as contributor

File: Assets/GameAssets/Boss/BossScripts/Attacks/AttackWarningZone.cs [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+        _warningMeshRenderer = _attackIndicator.GetComponent<MeshRenderer>();
+        _warningMeshRenderer.enabled = false;
+    }
+
+    /// <summary>
+    /// Starts the warning
+    /// </summary>
+    public void StartWarningZone()
+    {
+        print("Start warning zone");
```
Comment (ASLH68) [inline Assets/GameAssets/Boss/BossScripts/Attacks/AttackWarningZone.cs:77]: Do we need this print here?

File: Assets/GameAssets/Boss/BossScripts/Attacks/Vine/VineAttack.cs [hunk 3]
```diff
@@ -0,0 +1,132 @@
+/*****************************************************************************
+// File Name :         VineAttack.cs
+// Author :            Ryan Swanson
+// Creation Date :     11/7/2024
+//
+// Brief Description : Controls the functionality for the bosses vine attack
```
Comment (ASLH68) [inline Assets/GameAssets/Boss/BossScripts/Attacks/Vine/VineAttack.cs:6]: Update the description on `VineAttack.cs` and `VineAttackController.cs` so it's clear how they differ

**Review by ASLH68 — CHANGES_REQUESTED** [review]

**Review by ASLH68 — APPROVED** [review]

**Pull request merged**